src/SAWScript/Interpreter.hs

```diff
@@ -1830,6 +1830,19 @@ primitives = Map.fromList
     , "read-only regions."
     ]
 
+  , prim "crucible_alloc_readonly_aligned" "Int -> LLVMType -> CrucibleSetup SetupValue"
+    (bicVal crucible_alloc_readonly_aligned)
+    Current
+    [ "Declare that a read-only memory region of the given type should be"
+    , "a Crucible specification, and also specify that the start of the region"
+    , "should be aligned to a multiple of the specified number of bytes (which"
+    , "must be a power of 2). The specified alignment must be no less than the"
+    , "minimum required by the LLVM type. The function must not attempt"
+    , "to write to this memory region. Unlike `crucible_alloc`, regions"
+    , "allocated with `crucible_alloc_readonly` are allowed to alias other"
```

Comment:
Wrong function name

Reply:
The sentence "The specified alignment must be no less than the minimum required by the LLVM type" should be removed, as we are relaxing this requirement. (Similarly for the help text for `crucible_alloc_aligned`.)